### PR TITLE
finage crypto endpoint, WS support for crypto endpoint

### LINKED ISSUE #1269: Upgrade Finage (endpoint: crypto) to support websockets
Add crypto endpoint WS support for Finage

request: 
```{"action": "subscribe", "symbols": "REQ/USD"}```

output: 
`Value to use is 'p' in the returned json`

### PR DESCRIPTION
**Pull request by karen-stepanyan**
## Changes

- Added new endpoint for crypto prices.
- Added WS support for crypto endpoint
- Added integration test for crypto endpoint

## Steps to Test

`yarn test packages/sources/finage/test
`
## Quality Assurance

- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

File: packages/sources/finage/README.md [hunk 1]
```diff
@@ -2,19 +2,20 @@
...
 The adapter takes the following environment variables:
 
-| Required? |          Name           |                     Description                     | Options |            Defaults to            |
-| :-------: | :---------------------: | :-------------------------------------------------: | :-----: | :-------------------------------: |
-|    ✅     |        `API_KEY`        |                                                     |         |                                   |
-|           | `STOCK_WS_API_ENDPOINT` | The Websocket endpoint to connect to for stock data |         | `wss://e4s39ar3mr.finage.ws:7002` |
-|           | `FOREX_WS_API_ENDPOINT` | The Websocket endpoint to connect to for forex data |         | `wss://w29hxx2ndd.finage.ws:8001` |
+| Required? |           Name           |                     Description                      | Options |            Defaults to            |
+| :-------: | :----------------------: | :--------------------------------------------------: | :-----: | :-------------------------------: |
+|    ✅     |        `API_KEY`         |                                                      |         |                                   |
```
Comment (cds95) [inline packages/sources/finage/README.md:15]: Can we add `WS_SOCKET_KEY` to the ReadMe?

File: packages/sources/finage/schemas/env.json [hunk 2]
```diff
@@ -14,6 +14,10 @@
     "FOREX_WS_API_ENDPOINT": {
       "type": "string",
       "default": "wss://w29hxx2ndd.finage.ws:8001"
+    },
+    "CRYPTO_WS_API_ENDPOINT": {
+      "type": "string",
```
Comment (cds95) [inline packages/sources/finage/schemas/env.json:19]: Can we add `WS_SOCKET_KEY` here as well?

**Review by cds95 — CHANGES_REQUESTED** [review]
Looks good overall.  Left some minor comments.

**Review by austinborn — COMMENTED** [review]
LGTM, pending addition of `WS_SOCKET_KEY` env var to readme

File: packages/sources/finage/README.md [hunk 3]
```diff
@@ -13,6 +13,7 @@ The adapter takes the following environment variables:
 | Required? |           Name           |                     Description                      | Options |            Defaults to            |
 | :-------: | :----------------------: | :--------------------------------------------------: | :-----: | :-------------------------------: |
 |    ✅     |        `API_KEY`         |                                                      |         |                                   |
+|           |     `WS_SOCKET_KEY`      |                                                      |         |                                   |
```
Comment (cds95) [inline packages/sources/finage/README.md:16]: Can you add a description for it as well?

**Review by cds95 — DISMISSED** [review]

**Review by austinborn — APPROVED** [review]

**Review by cds95 — APPROVED** [review]

**Pull request merged**